openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -112,6 +112,15 @@ def process(self, instance):
         else:
             preset["viewport_options"] = {"imagePlane": image_plane}
 
+        # Image planes do not update the file sequence unless the active panel
+        # is viewing through the camera.
+        panel_camera = cmds.modelPanel(
+            instance.data["panel"], query=True, camera=True
+        )
+        cmds.modelPanel(
+            instance.data["panel"], edit=True, camera=preset["camera"]
+        )
+
```

Comment:
Does this also make sense to be done through a contextmanager?

```python
import contextlib
from maya import cmds

@contextlib.contextmanager
def panel_camera(panel, camera):
    original_camera = cmds.modelPanel(panel, query=True, camera=True)
    try:
        cmds.modePalen(panel, edit=True, camera=camera)
        yield
    finally:
        cmds.modePalen(panel, edit=True, camera=original_camera)


with panel_camera(instance.data["panel"], preset["camera"]):
    # playblasting here
```

Additionally - should any of this be done for thumbnails as well? Or is it just that they don't update over time?

Reply:
Yup will have a look later.

Reply:
Done